src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,343 @@
+package duke;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class Duke {
+
+    public static void main(String[] args) {
+        showWelcomeScreen();
+
+        ArrayList<Task> storeTasks = new ArrayList<>();
```

Comment:
Maybe consider renaming storeTasks to taskList, task-<Noun> etc., like the format of tasksText in line 17.